### ISSUE TITLE
Want to set default font size to use in build-in transforms.

### ISSUE THREAD

**Issue by funyapu** [amzn/style-dictionary#373]
All of the build-in rem converting transforms uses 16 as base px.
It would be nice if user can set their default value for better flexibility.

**Comment by dbanksdesign** [comment]
This is a great suggestion! Would setting an attribute on the platform object work? Like:

```
{
  "source": ["tokens/**/*.json"],
  "platforms": {
    "web": {
      "baseFontSize": 14,
      "transformGroup": "web",
      "files": [{ }]
```

**Comment by funyapu** [comment]
Thanks. Seems brilliant to me.

One thing people may want in additional is having multiple base font size per screens.
Many websites and apps are having larger font size in mobile view.

**Comment by dbanksdesign** [comment]
This was solved in #505 and is available in 3.0 which you can get a pre-release of with `npm install style-dictionary@next`
